FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentButton.cs

```diff
@@ -31,6 +31,9 @@ public bool IsChecked {
     [MemberFunction("E8 ?? ?? ?? ?? 48 8D 05 ?? ?? ?? ?? 88 9F ?? ?? ?? ??")]
     public partial void Ctor();
 
+    [MemberFunction("48 89 5C 24 ?? 57 48 83 EC 20 48 8B D9 48 8B FA 48 8B 89 ?? ?? ?? ?? 48 85 C9 0F 84 ?? ?? ?? ?? 48 85 D2 74 ?? 0F B6 81"), GenerateStringOverloads]
+    public partial void SetText(byte* text);
```

Comment:
This function sets the text pointer to what is sent in here and can cause memory corruption with `GenerateStringOverloads` as it calls `AtkTextNode.SetText` inside this function

Ref: #1040

Reply:
Thanks. Removed the attribute.